teamworkApp/lib/generateTeams.py

```diff
@@ -1,4 +1,8 @@
-import argparse 
+# muddersOnRails()
+#
```

Comment:
Do we want a header on this file?

Reply:
I didn't have enough information for this file to fill in the rest. Do you? or would Reagan or Nathan?